R/generate_template.R

```diff
@@ -12,6 +12,7 @@
 #' @param name name for the new application and directory
 #' @param location base path for creation of \code{name}
 #' @param sampleapp whether to create a sample shiny application
+#' @param right_sidebar parameter to set the right_sidebar. It can be TRUE/FALSE or a custom icon. See ?shiny::icon() for how to create an icon.
```

Comment:
need to hard-wrap this line to match the others.

Reply:
Move the See ?shiny::icon() for how to create an icon to the information section (see below).  You also should link to the shiny::icon help location in a see also `@seealso \link[shiny:icon]{shiny:icon()}`

Reply:
Still outstanding - both on the word wrap and the seealso addition.